Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -0,0 +1,186 @@
+# Implement a Tracer
+
+## Overview
+
+This guide is aimed at ``TracerProtocol`` and ``InstrumentProtocol`` implementation authors.
+
+This guide is for you if you find yourself in need of implementing your own tracing client such as Zipkin, Jaeger, X-Trace, OpenTelemetry or something similar that is custom to your company or distributed system.
+
+## Do you need an Instrument or a Tracer?
+
+Distributed tracing offers two types of instrumentation protocols: an instrument, and a tracer.
+
+A tracer is-an instrument as well, and further refines it with the ability to start a trace ``Span``.
```

Comment:
```suggestion
A tracer is an instrument as well, and further refines it with the ability to start a trace ``Span``.
```

Reply:
I'm seeing this still in the diff, maybe this comment shouldn't be resolved?